plugins/woocommerce-admin/client/layout/footer/footer.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { useSlot } from '@woocommerce/experimental';
+
+/**
+ * Internal dependencies
+ */
+import './footer.scss';
+import { WC_FOOTER_SLOT_NAME, WooFooterItem } from './utils';
+
+export const Footer: React.FC = () => {
```

Comment:
A simple Footer component with a SlotFill, this allows us to easily fill it with the CES footer and correctly use the full width.

Reply:
💯